src/modules/eventRegister/personal-info-des/PersonalInfoDes.vue

```diff
@@ -31,23 +31,23 @@
       <div class="flex flex-col w-1/3 mr-4">
         <div>Student ID</div>
         <BaseTextInput
-          :value="user.chulaId"
+          :value="user && user.chulaId"
           class="w-full h-4 disabled: bg-white text-primary-6"
           disabled
         />
       </div>
       <div class="flex flex-col w-1/3 mr-4">
         <div>Faculty</div>
         <BaseTextInput
-          :value="faculty"
+          :value="user && faculty"
```

Comment:
Note: Will need backend to implement faculty.

Reply:
correction: Consulted with @trif0lium  I will do it on frontend

Reply:
Correction: After consulted with @trif0lium, It is easier to do it on frontend.